frontend/src/models/dashboardsModel.tsx

```diff
@@ -237,19 +238,17 @@ export const dashboardsModel = kea<dashboardsModelType>({
         nameSortedDashboards: [
             () => [selectors.rawDashboards],
             (rawDashboards) => {
-                return [...Object.values(rawDashboards)].sort((a, b) =>
-                    (a.name ?? 'Untitled').localeCompare(b.name ?? 'Untitled')
-                )
+                return [...Object.values(rawDashboards)]
+                    .filter((dashboard) => !(dashboard.name ?? 'Untitled').startsWith(GENERATED_DASHBOARD_PREFIX))
```

Comment:
Wait, if the ones starting with `GENERATED_DASHBOARD_PREFIX` are filtered out, it doesn't make sense to sort by that prefix, right?

Reply:
Not here, but, this isn't the only place the comparison function is used, and I wanted to keep the function standard for sorting wherever you want to sort dashboards, so if, say, you're sorting raw dashboards somewhere, they're still at the end. Seemed less error prone.

Happy to remove generated prefix from the sorting function if you feel strongly about this

Reply:
Right, but `nameSortedDashboards` is the source for all our lists of dashboards, so if we filtered out the generated dashboards, they won't be displayed at the bottom anymore – they will be just hidden. From the description of the PR I thought that that's not the intention

Reply:
original intention yes, but based on Li's comment, am removing them completely from the list, and adding a link to the usage dashboard to get to the main db, will update description 😅

Reply:
Oh, I missed that, was still looking at the PR title 😅 In that case `nameCompareFunction` handling is unused in this PR, so would be better to revert that addition

Reply:
I'd argue that it at least unifies the current 3 places where we manually write the same function again, would you be fine with me removing the generated prefix from it but keeping the function, or do you want me to get rid of it completely?

Reply:
No, yeah, the function makes sense. Just the prefix handling that's not going to be used